substrate/frame/fast-unstake/src/benchmarking.rs

```diff
@@ -116,21 +119,22 @@ benchmarks! {
 				..
 			}) if checked.len() == 1 && stashes.len() as u32 == b
 		));
-	}
-	: {
-		on_idle_full_block::<T>();
-	}
-	verify {
-		assert!(matches!(
+
+		#[block]
+		{
+			on_idle_full_block::<T>();
+		}
+
+		assert_eq!(
 			fast_unstake_events::<T>().last(),
-			Some(Event::BatchFinished { size: b })
-		));
+			Some(&Event::BatchFinished { size: b })
+		);
```

Comment:
`b` was ignored in the previous statement, but the unused warning didn't show in the bench v1 syntax.

Now the value is actually asserted.